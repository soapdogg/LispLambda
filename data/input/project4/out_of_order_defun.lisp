
(DIFF 5 6)
(DEFUN DIFF (X Y)
    (COND ((EQ X Y) NIL) (T T)))

(DIFF (+ 1 1) 4)
(DIFF T NIL)
(DIFF T T)
(DIFF (* (+ 1 3) 2) (- 12 4))