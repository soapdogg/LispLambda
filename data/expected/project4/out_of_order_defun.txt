T
T
T
NIL
NIL